decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -10,11 +10,20 @@ module LayoutHelper
     def favicon
       return if current_organization.favicon.blank?
 
-      safe_join(Decidim::OrganizationFaviconUploader::SIZES.map do |version, size|
-        favicon_link_tag(current_organization.attached_uploader(:favicon).variant_url(version, host: current_organization.host), sizes: "#{size}x#{size}")
+      safe_join(Decidim::OrganizationFaviconUploader::SIZES.map do |version, _|
```

Comment:
I have a doubt about the current implementation. What I see is that we have this:

```html

<link rel="shortcut icon" type="image/x-icon" href="http://localhost/.../logo.png" sizes="any"><link rel="shortcut icon" type="image/x-icon" href="http://localhost/.../logo.png" sizes="any"><link rel="shortcut icon" type="image/x-icon" href="http://localhost/.../logo.png" sizes="any">
<link rel="apple-touch-icon" type="image/x-icon" href="http://localhost/.../logo.png" sizes="180x180">
<link rel="manifest" href="/manifest.webmanifest">
```

So we have 3 shortcut icons with the different sizes, but all have the `any` size attribute. As far as I understood the [evilmartians.com guide](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) we should only have one resized to 32x32 with "sizes=any". I don't know how browsers will behave if they have 3 same tags, is there any reason for doing it like that? Maybe we should just leave it as it was, with the different sizes explicit? Or just leave one with 32x32 as the guide says? What do you think? (Who would say that a freaking icon has that many troubles 😅)